src/AzureExtension/DevBox/DevBoxInstance.cs

```diff
@@ -128,14 +178,20 @@ private async Task<(Uri? WebURI, Uri? RdpURI)> GetRemoteLaunchURIsAsync(Uri boxU
     }
 
     public ComputeSystemOperations SupportedOperations =>
-        ComputeSystemOperations.Start | ComputeSystemOperations.ShutDown | ComputeSystemOperations.Delete;
+        ComputeSystemOperations.Start | ComputeSystemOperations.ShutDown | ComputeSystemOperations.Delete | ComputeSystemOperations.Restart;
 
     public bool IsValid
     {
         get;
         private set;
     }
 
+    public string AlternativeDisplayName => new string("Project : " + ProjectName ?? "Unknown");
+
+    public IDeveloperId AssociatedDeveloperId => throw new NotImplementedException();
+
+    public string AssociatedProviderId => throw new NotImplementedException();

```

Comment:
We can add the ComputeSystemProvider's Id to this before sending it back to Dev Home

Reply:
Changed.